src/main.cpp

```diff
@@ -0,0 +1,10 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#include <NovelRT.h>
+
+int main(int argc, char *argv[])
+{
```

Comment:
bracing style is off here, should probably be K&R as opposed to Allman.

Reply:
Fixed. Force of habit 😄